portability-types-common/src/main/java/org/datatransferproject/types/common/models/photos/PhotoModel.java

```diff
@@ -58,6 +60,7 @@ public PhotoModel(
     this.dataId = dataId;
     this.albumId = albumId;
     this.inTempStore = inTempStore;
+    this.sha1 = sha1;
```

Comment:
So that we dont have to modify the tests/exporters for Koofr And Facebook, can we add another constructor here that preserves this behavior of having `uploadedTime` without `sha1`? that would make this PR easier to read imo